app/helpers/react_components/donations/footer_form.rb

```diff
@@ -17,7 +17,7 @@ def to_s
             recaptcha_site_key: ENV.fetch('RECAPTCHA_SITE_KEY', Exercism.secrets.recaptcha_site_key),
             links: {
               settings: Exercism::Routes.donations_settings_url,
-              donate: Exercism::Routes.donate_url
+              confirm_params_return_url: Exercism::Routes.donate_url
```

Comment:
This goes into `stripe.confirmPayments.options.confirmParams.returnUrl`

Reply:
I would probably rename it slightly to something like `donate_return_url`.

Reply:
I think I'm ok with the `confirm_params_return` bit as it's in the scope of the form, but agree it probably shouldn't have the `_url` prefix. Pragmatically I don't care if its hard to change it, but if it's just a couple of renames it would be better to be consistent.

Reply:
I agree, but the reason I chose this name is because this is exactly what the API accepts. so it is pretty specific.

Reply:
Well, the reason I was suggesting `donate_return_url` (actually, `redirect_to_after_donation` or something like that might be better) is because it isn't coupled to Stripe. I feel that's an implementation detail 🤷